docs/docusaurus.config.ts

```diff
@@ -13,7 +13,7 @@ const ingoPreset = {
   docs: {
     showLastUpdateAuthor: false,
     showLastUpdateTime: true,
-    includeCurrentVersion: false,
+    includeCurrentVersion: true,
```

Comment:
Update the [CI](https://github.com/ingonyama-zk/icicle/blob/main/.github/workflows/build-docs.yml#L22) to set `process.env.NODE_ENV` to "production";
```suggestion
    includeCurrentVersion: process.env.NODE_ENV !== "production",
```